perfkitbenchmarker/providers/gcp/util.py

```diff
@@ -113,3 +113,61 @@ def _AddCommonFlags(self, resource):
     if hasattr(resource, 'zone'):
       self.flags['zone'] = resource.zone
     self.additional_flags.extend(FLAGS.additional_gcloud_flags or ())
+
+
+class GCPLocation(object):
+  """A GCP location.
+
+  This class may represent a zone, region, or continent.
```

Comment:
Oops, left over from an old version. "Multiregions" were initially "continents" until I renamed them. Looks like I missed this reference.